package.json

```diff
@@ -9,7 +9,7 @@
     "licenses": "yarn licenses generate-disclaimer > 3RD_PARTY_LICENSES",
     "markdownlint": "markdownlint *.md ./github/**/*.md",
     "shellcheck": "shellcheck .github/workflows/*.sh",
-    "test": "node test/setup-beam.test.js",
+    "test": "node --test --test-reporter=spec",
```

Comment:
Without `--test-reporter=spec` `::debug::` gets output as `# ::debug::` which means GitHub Actions can't understand debug is off.